docs/kb/kb0004-akv2aks.md

```diff
@@ -4,257 +4,156 @@ title: Sync certificates and secrets from Azure Key Vault to Azure Kubetnetes Se
-  containers:
-  - name: busybox
-    image: k8s.gcr.io/e2e-test-images/busybox:1.29
-    command:
-      - "/bin/sleep"
-      - "10000"
-    volumeMounts:
-    - name: secrets-store-inline
-      mountPath: "/mnt/secrets-store"
-      readOnly: true
-  volumes:
-    - name: secrets-store-inline
-      csi:
-        driver: secrets-store.csi.k8s.io
-        readOnly: true
-        volumeAttributes:
-          secretProviderClass: "azure-key-vault-name" # the name of your key vault
-```
-
-```powershell
-kubectl apply -f ./test-pod.yml
-
-kubectl exec busybox-secrets-store-inline -- ls /mnt/secrets-store/
-# Expected output:
-# cert-demo
-# foo
-
-kubectl exec busybox-secrets-store-inline -- cat /mnt/secrets-store/foo
-# Expected output:
-# bar
-
-kubectl exec busybox-secrets-store-inline -- cat /mnt/secrets-store/cert-demo
-# Expected output:
------BEGIN PRIVATE KEY-----
-MIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQDVYhtyud6rbRJT
-...
-3fic6VM3cQR9FJxBxAq4vro=
------END PRIVATE KEY-----
------BEGIN CERTIFICATE-----
-MIIDQjCCAiqgAwIBAgIQSRZYP7ncTSGCw6IEOxTIhjANBgkqhkiG9w0BAQsFADAe
-...
-5STNJyO/kEBkBMjlzZKlDkhuf4Tr1g==
------END CERTIFICATE-----
-```
-
-```powershell
-kubectl get secrets
-# Expected output:
-# NAME                                    TYPE                                  DATA   AGE
-# cert-demo                               kubernetes.io/tls                     2      9h
-# foo                                     Opaque                                1      9h
-```
+    In your values file, we'll need to add the following block of code into the global values:
+
+    ```yaml
+    global:
+      keyvault:
+        enabled: true # When enabled, it will deploy the secret store manifest to Kubernetes
+        #frontendCrt: cluedin-sample-pfx # This must match the certificate name on the AKV end. When mounted, it will appear as `cluedin-frontend-crt`
+        userAssignedIdentityID: $kvManagedIdentity # This is the guid for the kv managed identity
+        keyvaultName: $kv.name # This is your key vault name
+        tenantId: $tenantId # This is the guid of your tenant
+        secretRanges:
+        - secretName: cluedin-login-details # This is how the secret will appear within Kubernetes Secrets once synchronised
+          secretType: Opaque # For most secrets, leave as Opaque
+          secretKeys:
+            password: cluedin-login-password # The left side (Key) is the name in the Kubernetes secret object. The right side (Value) is the AKV reference which will grab its value.
+            username: cluedin-login-username
+
+    infrastructure:
+      cert-manager:
+        #enabled: false # Only set to false if using an uploaded frontend certificate.
+    ```
+    **NOTE**: If you are using the certificate upload as part of your setup, you **must** disable cert-manager by setting `enabled: false`. You also must set `frontendCrt` to a value. It will be mounted as `cluedin-frontend-crt`. This secret name cannot change.
```

Comment:
```suggestion
    {:.important}
    If you are using the certificate upload as part of your setup, you must disable cert-manager by setting `enabled: false`. You also must set `frontendCrt` to a value. It will be mounted as `cluedin-frontend-crt`. This secret name cannot change.
```